solr/README.md

```diff
@@ -0,0 +1,112 @@
+# solr 
+
+This folder contains the solr schema and configuration for the PLOS journals
+stack. 
+
+# Cloud Configuration 
+
+## Overview
+
+A solr configuration consists of a named hierarchy of files. Each collection on
+a cluster must use a configuration that has been uploaded to solr. Many
+configurations can be stored, but only one can be used per collection.
+
+The configurations for solr cloud are stored in zookeeper. The easiest way to
+manipulate them is to use the `zkcli.sh` script bundled with solr. (Note that solr's 
+`zkcli.sh` is NOT the same as the `zkCli.sh` that comes bundled with zookeeper.) 
+The easiest way to run `zkcli.sh` is via `docker run`.
+
+For example ... 
+
+```
+docker run --rm -v ~/repos/plos/wombat/solr:/solrconf solr:7.7 \
+./server/scripts/cloud-scripts/zkcli.sh -z solr-zoo-201.soma.plos.org:2181/solr-mega \
+-cmd upconfig -confdir /solrconf/config -confname wombat_31d72ab_2020-06-22
+```
+
+You can also download configurations and perform a handful of other useful
+tasks with `zkcli.sh`. To see all the available options:
+
+```
+docker run --rm solr:7.7 ./server/scripts/cloud-scripts/zkcli.sh help
+```
+
+If you don't want to use docker for some reason, you can download the solr
+release and run `zkcli.sh` from there. 
+
+
+## Naming Conventions
+
+When uploading a configuration to solr, name it after the repo and hash where
+the configuration can be found in version control, along with the date of
+creation. This will make it easy to see at glance which configuration is
+newest, while also providing an easy reference to the source code where the
+config can be located. For configurations deployed from this repo, the name in
+solr should generally look something like `wombat_31d72ab_2020-06-22`.
```

Comment:
Can we move the date in front of the hash? That way alphanumeric sort gives us a chronological sort

Reply:
Good call @JackLaBarba, updated accordingly.